README.md

```diff
@@ -1,64 +1,199 @@
-# CrateDB + Apache Spark
+# Machine Learning with CrateDB and Apache Spark
 
-CrateDB is a distributed SQL database system which is capable of managing huge amounts of data in real time.
-CrateDB provides an easy, horizontal scalable and system independent solution built for IoT applications.
-This project explains the necessary steps to combine CrateDB with Apache Spark, a lightning fast cluster system which offers a variety operations for BigData.
-A really poplar feature of Spark is machine learning, which will be covered here.
-This project explains the necessary steps to apply machine learning algorithms on big data.
-For storage and ingestion, CrateDB is the software to store and query an incredible amount of data in real time.
-For machine learning, Apache Spark is the quick and easy way to produce machine learning models.
+## About
 
+[CrateDB](http://crate.io) is a distributed SQL database system which is capable of managing huge amounts of data in real time.
+It is an easy, horizontal scalable, efficient and system independent solution built for IoT applications.
+
+Though CrateDB is a pretty solid choice for managing BigData, it was not built for performing general data analysis.
```

Comment:
You could do data analysis in a basic way (min, max, avg,...). Would say something like 

> ... for advanced data processing that exceeds the capabilities of standard SQL.